.github/workflows/test-linux-gpu.yml

```diff
@@ -121,19 +121,29 @@ jobs:
         if: ${{ steps.setup.outputs.build-type == 'Debug' }}
         run: |
           mkdir coverage
-          lcov --capture --directory build/src --output-file coverage/coverage.info
-          lcov --remove coverage/coverage.info "*/c++/*" "*/boost/*" "*/usr/*" "*/deps/*" "*/src/mal/*" --output-file coverage/coverage.f.info
-          genhtml coverage/coverage.f.info --output-directory coverage/output
-          bash <(curl -s https://codecov.io/bash) -f coverage/coverage.f.info || echo "Codecov did not collect coverage reports"
+          lcov \
+            --capture \
+            --directory build/src \
+            --output-file coverage/coverage.info
+          lcov \
+            --remove coverage/coverage.info "*/c++/*" "*/boost/*" "*/usr/*" "*/deps/*" "*/src/mal/*" \
+            --output-file coverage/coverage.linux-gpu.info
       - name: Upload coverage (Debug)
         if: ${{ steps.setup.outputs.build-type == 'Debug' }}
         uses: actions/upload-artifact@v3
         with:
           name: shards-linux-gpu-coverage
           path: |
-            coverage
-            test-gfx.xml
+            coverage/coverage.linux-gpu.info
           if-no-files-found: error
+      # - name: Test results (Debug)
```

Comment:
Remove comment

Reply:
I kept it there on purpose. It was uploading a file `test-gfx.xml` but I didn't want to mess up the coverage artifacts so I disabled it.

If you say we don't need it anymore, I can indeed remove this commented part.

Reply:
Ah yeah if it's just the xml, you can add it to the `if: failure()` step below, it's more useful there anyway.

In the case a graphics comparison tests fails it'll upload the images there and then the xml can be there too containing a list of which tests failed/passed